charts/newrelic-pixie/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
does it make sense to bump this to v2 or we want to support Helm 2 as well?

Updating this is in the future might be more painful

Reply:
This is the api version of the chart itself. I can update it. 👍

Reply:
but v2 works with helm3 only I believe

Reply:
@paologallinaharbur in fact, it's better to keep api version 1 here. We still need to support Helm 2 and Helm 3 is compatible with with this api version. We should only move when we know that Helm 2 isn't supported anymore or that there's a Helm 4 that isn't compatible with api version 1.